### DISCUSSION TITLE
You unlocked new Achievements on GitHub! Show them off by updating your Profile settings

### DISCUSSION THREAD

**Discussion by Tyriar** [community/community#19634] [Other Feature Feedback, Questions, & Ideas]
I got this message when I opened my profile:

![image](https://user-images.githubusercontent.com/2193314/176221271-14532e6a-a028-4f6a-b5c5-9e24799065b2.png)

I don't understand why it's there because:

a. I explicitly disabled the feature[^1], which is a clear signal I don't want it to bother me
b. I didn't actually unlock new achievements?

[^1]: Because it takes up too much room, especially on mobile, and I don't feel they add anything of value

**Comment by rodukov** [comment]
* If you do not want to show this notify just click on X
* You can enable achievements in settings and watch, whats you unlocked

**Reply by Tyriar** [reply]
Clearly I can click the x, but I should not be notified at all because I explicitly disabled the feature.

**Comment by JhonnyLusonode** [comment]
@Tyriar seems like might be a bug to me or maybe it´s something that github forgot to change.

You should report it to [Github Bug Report](https://support.github.com/contact/bug-report).

Hope this clarifies.

If this reply is helpful, i would ask you to 'Mark as answer' so i can get the achievement pls. Thanks!
